packages/components/psammead-story-promo/src/index.jsx

```diff
@@ -177,8 +171,8 @@ const TextGridFallbackTopStory = css`
     padding: 0 ${GEL_SPACING_DBL};
   }
 
-  @media (min-width: ${GEL_GROUP_5_SCREEN_WIDTH_MIN}) {
-    width: ${eightOfTwelveColumnsMaxScaleable};
+  @media (min-width: ${GEL_GROUP_4_SCREEN_WIDTH_MIN}) {
+    width: ${halfWidthColumnsMaxScaleable};
```

Comment:
Should this also be using `calc`?

Reply:
I don't think so since we are not performing any calculation.

Reply:
You are right. it is a `TextGridFallback`